xxx/build.gradle

```diff
@@ -21,8 +20,8 @@ repositories {
 }
 
 dependencies {
-    compile('org.springframework.boot:spring-boot-starter-webflux')
-    testCompile('org.springframework.boot:spring-boot-starter-test')
+    compile("org.springframework.boot:spring-boot-starter-webflux:${springBootVersion}")
+    testCompile("org.springframework.boot:spring-boot-starter-test:${springBootVersion}")
```

Comment:
That's not right, it does. Maybe @wilkinsona knows what the problem is on your project.